app/backend/src/couchers/servicers/api.py

```diff
@@ -194,6 +216,12 @@ def UpdateProfile(self, request, context):
             if request.languages.exists:
                 user.languages = "|".join(request.languages.value)
 
+            if request.language_abilities:
+                user.language_abilities = [
+                    LanguageAbilities(language=l.code, fluency=fluency2sql[l.fluency])
+                    for l in request.language_abilities.value
+                ]
+
```

Comment:
Does this automatically remove these when they're removed from abilities? Please add a test to check.

Reply:
Great catch, It doesn't, the test fails 😬 

```
E       sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) null value in column "user_id" of relation "language_abilities" violates not-null constraint
E       DETAIL:  Failing row contains (1, eng, native, null).
```